### PR TITLE
In d aadhaar data masking

### PR DESCRIPTION
**Pull request by gowthamd3**
---
When submitting a connector, please check the following conditions for your PR.

- [ ] I attest that the connector works and I verified by deploying and testing all the operations.
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`.

### CODE REVIEW

File: certified-connectors/IN-D Aadhaar Number Masking/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# IN-D Aadhaar Number Masking
```
Comment (sriyen-msft) [inline certified-connectors/IN-D Aadhaar Number Masking/README.md:1]: @gowthamd3 Please change the title of your connector to `IN-D Aadhaar Data Masking` here in the readme.md and anywhere in the connector files

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@gowthamd3 Please sync your local branch with remote repo (dev) to resolve the branch conflicts

File: certified-connectors/IN-D Aadhaar Number Masking/README.md (hunk 1 above)
Comment (gowthamd3) [reply certified-connectors/IN-D Aadhaar Number Masking/README.md]: i have updated readme.md file

Comment (gowthamd3) [reply certified-connectors/IN-D Aadhaar Number Masking/README.md]: @sriyen-msft, updated redme.md file

**Comment by sriyen-msft** [comment]
@gowthamd3 I am cancelling this PR as there is no response from the partner

**Pull request closed without merging**